pygfx/renderers/wgpu/meshshader.py

```diff
@@ -136,6 +150,19 @@ def get_bindings(self, wobject, shared):
                 material, "env_mapping_mode", "CUBE-REFLECTION"
             )
 
+        # set lightmap configs
+        if getattr(material, "light_map", None):
+            if "use_texcoords1" not in self.kwargs:
+                warnings.warn(
+                    "Light map requires a second set of texture coordinates (geometry.texcoords1), but none were found, so it will be ignored."
+                )
```

Comment:
I'm not sure where we stand currently with respect to warnings @almarklein ? Do we already use them in the codebase?

If not, I would suggest raising a `ValueError` instead.

Reply:
I agree this should be a `ValueError`.

Reply:
re warnings:  #584